ci/closures.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -xe
+
+# Env variables needed for `beaver dlang` commands
+export DMD=2.071.2.s*
+export F=devel
+# Enables printing of all potential GC allocation sources to stdout
+export DFLAGS=-vgc
+
+# Prepare docker image and sources
+beaver dlang install
+beaver dlang make d2conv
+
+# Ensure that there are no lines about closure allocations in the output
+! 2>&1 beaver dlang make fasttest | grep "closure"
```

Comment:
Just cosmetic, but the `!` gets a bit lost if you do the redirection at the start, why not
```sh
! beaver dlang make fasttest 2>&1 | grep "closure"
```

Reply:
Sure, no preferences here.